forum/forum/urls.py

```diff
@@ -67,4 +67,6 @@ def api_test(request,format=None):
     path('',test,name='test'),
     path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
     path('redoc/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
+    re_path(r'^auth/', include('djoser.urls')),
```

Comment:
Everything above seems legit, the only thing is idk if other links beside djoser.urls.jwt are necessary, but idk